utils/Header/Header.style.js

```diff
@@ -0,0 +1,23 @@
+import { styled } from "styled-components";
+
+export const StyledDiv = styled.div`
+  background-color: var(--mainDarkColor);
+  color: var(--mainLightColor);
+  position: fixed;
+  top: 0;
+  margin: 0;
+  width: 100%;
+  z-index: 1;
+`;
+
+export const StyledH1 = styled.h1`
```

Comment:
Same as above 😬

Reply:
I tried to make the semantic more selfexplaining